test/lsp-clients-test.el

```diff
@@ -69,6 +69,18 @@
   (let ((major-mode 'js-mode))
     (should (lsp-clients-flow-activate-p (concat test-location "fixtures/SampleFlowProject/src/sample.js") nil))))
 
+(ert-deftest lsp-flow-should-activate-on-flow-project-without-flow-file-comment ()
+  ;; Set `js-mode' ON and check that a Flow project activates the Flow but without flow tag
```

Comment:
I think the test name is self-explanatory and we can just remove this comment. The same for the other test.